wp-content/themes/core/components/statistic/Controller.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Tribe\Project\Templates\Components\statistic;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+
+/**
+ * Class Statistic
+ */
+class Controller extends Abstract_Controller {
+	/**
+	 * @var string
+	 */
+	public $tag;
+	/**
+	 * @var string[]
+	 */
+	private $classes;
+	/**
+	 * @var string[]
+	 */
+	private $attrs;
+	/**
+	 * @var Deferred_Component
+	 * @uses components/text
+	 */
+	private $value;
+	/**
+	 * @var Deferred_Component
+	 * @uses components/text
+	 */
+	private $label;
+
+	public const TAG     = 'tag';
```

Comment:
Couple of things.
- Let's move the constants above properties.
- We're naming Controllers to be more descriptive now. So `Statistics_Controller`
- We're doing typed properties, which is new in php 7.4. See https://github.com/moderntribe/square-one/blob/epic/fidgety-feet/wp-content/themes/core/components/blocks/logos/Logos_Block_Controller.php#L25-L33